docs/sources/administration/configuration.md

```diff
@@ -1473,12 +1473,10 @@ Enter a comma-separated list of plugin identifiers to identify plugins that are
 
 ### plugin_admin_enabled
 
-
 Available to Grafana administrators only, the plugin admin app is set to `false` by default. Set it to `true` to enable the app.
 
 For more information, refer to [Plugin catalog]({{< relref "../plugins/catalog.md" >}}).
 
```

Comment:
Don't forget to add **plugin_admin_remote_manage_enabled here.

Reply:
See note in the PR description
>NOTE: I have intentionally omitted the new config from configuration.md as this config is only relevant for HG which will be manually controlled. Users won't ever need/want to manually this manually

Reply:
If the option is visible to all users in the .ini docs, then we need to define it, even if all we say is, "Only applicable to Grafana Cloud users."